### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by aljones15**
To Do:

- [x] Move oath2 up in scope
- [x] Refactor calls on issuer to use oath2 up a little
- [ ] use data integrity test suite on successful issue
- [x] possible add dynamic ids to issuer tests to prevent duplicate VC errors
- [ ] Test with an actual oath2 clientSecret (our oauth2 code is not quite the same as the interoperability stuff)

### CODE REVIEW

File: implementations/DigitalBazaar.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "Digital Bazaar",
+  "implementation": "Demo",
```
Comment (msporny) [inline implementations/DigitalBazaar.json:3]:
```suggestion
  "implementation": "Veres Issuer (Q/A)",
```

**Review by msporny — APPROVED** [review]
LGTM for first pass. Let's get this on main and see how it looks.
